### DISCUSSION TITLE
Your ideas should go here

### DISCUSSION THREAD

**Discussion by struckchure** [Dev-47/quiz-app#8] [Ideas]
Keep 'em coming folks

**Comment by horlakz** [comment]
There's not need for profile page, just dashboard is enough
Consist of average score in percent, number of quiz answered 
Then profile details normal, i think that may be all for now

**Comment by suftdev** [comment]
![Screenshot_20211231-140236.png](https://user-images.githubusercontent.com/73845970/147824815-d77baaee-c631-4e62-828d-668d84ed87d2.png)

Is this possible?

**Reply by horlakz** [reply]
Not yet

**Reply by struckchure** [reply]
exactly
